tests/integration/targets/custom_images/tasks/tests.yml

```
@@ -0,0 +1,184 @@
+---
+- name: Deleta a non existent custom image by uuid
+  cloudscale_ch.cloud.custom_images:
+    uuid: "{{ item }}"
+    state: absent
+  loop:
+    - '415cded5-d57c-4dc9-88fb-ececd38fb8ea'
+  ignore_errors: true
+
+- name: Deleta a non existent custom image by name
+  cloudscale_ch.cloud.custom_images:
+    name: this-image-is-non-existent
+    state: absent
+  ignore_errors: true
```

why the ignore errors? we don't expect an error here.